cockatrice/src/tab_game.cpp

```diff
@@ -274,13 +275,26 @@ void DeckViewContainer::sideboardLockButtonClicked()
 
 void DeckViewContainer::sideboardPlanChanged()
 {
+    qDebug() << "Standard Sideboard";
     Command_SetSideboardPlan cmd;
     const QList<MoveCard_ToZone> &newPlan = deckView->getSideboardPlan();
     for (int i = 0; i < newPlan.size(); ++i)
         cmd.add_move_list()->CopyFrom(newPlan[i]);
     parentGame->sendGameCommand(cmd, playerId);
 }
 
+void DeckViewContainer::sideboardPlanChanged(QList<MoveCard_ToZone> *newPlan)
+{
+    qDebug() << "Advanced Sideboard" << newPlan->size();
+    Command_SetSideboardPlan cmd;
+    for (int i = 0; i < newPlan->size(); ++i)
+        cmd.add_move_list()->CopyFrom(newPlan->at(i));
+    parentGame->sendGameCommand(cmd, playerId);
+
+    // TODO: Need to figure out how to re-paint the deck view
```

Comment:
[DeckViewScene::applySideboardPlan()](https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/deckview.cpp#L333) is probably a good starting point.
You probably want to handle this inside `DeckView::mouseDoubleClickEvent`, get the old and new `DeckViewCardContainer` zones from the scene()->cardContainers list, and then move the card from one zone to another the same way it's implemented in [DeckViewCardDragItem::handleDrop](https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/src/deckview.cpp#L36)